Update Promscale resource recommendation (#1925)
timescale/docs: promscale/recommendations/config-recomm.md

*   `checkpoint_timeout=15min` - when a lot of data is ingested, increase the
  checkpoint timeout to reduce the input/output pressure.
*   `max_wal_size` - set it to a high enough value so that the checkpoint is triggered
  by the timeout setting, much before the `max_wal_size` is reached.
*   `synchronous_commit=off` - this does not cause data corruption or
  inconsistency. However, in case of a crash, some of the most recent data points could be
  lost. For a monitoring observability use case, it's a reasonable trade-off to
  increase ingest performance.

> * `shared_buffers` - set it to at least 50% of available memory, we tried to estimate it for metrics ingestion,  see [Promscale Resource recommendations guide][promscale-resource-recommendations].

Reviewer: ```suggestion
* `shared_buffers` - set it to at least 50% of available memory and estimated based on metrics ingestion,  see [Promscale Resource recommendations guide][promscale-resource-recommendations].
```
